examples/main/package.json

```diff
@@ -10,15 +10,17 @@
     "d3-color": "^1.0.1",
     "d3-random": "^1.0.2",
     "d3-scale": "^1.0.3",
+    "deck.gl": "^4.0.0-rc.3",
     "immutable": "^3.8.1",
+    "luma.gl": "^3.0.0",
     "react": "^15.4.1",
+    "react-autobind": "^1.0.6",
     "react-dom": "^15.4.1",
-    "react-map-gl": "^1.7.2"
+    "react-map-gl": "^3.0.0-alpha.13"
```

Comment:
`^` doesn't work well for alpha releases, it will only install the specified version. `>=` is what we use in other repos for these situations.